server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java

```diff
@@ -78,11 +79,20 @@ public String processDatetimeParameter(String datetime, String filter) {
 
         // for now, assumption is that temporal is the only filter
         if (filter == null) {
-            if (datetime.startsWith("../") || datetime.startsWith("/")) {
+            if (datetime.startsWith("../")) {
```

Comment:
I think bug here as datetime = ../2023-10-23T00:00:00.000Z will not work with filter=temporal before ../2023-10-23T00:00:00.000Z,

I think it can be simplify like 

if(datetime.startsWith("../") || datetime.startsWith("/") {
      datetime = datetime.split("/")[1]   // Double check if "/" works or need "\/"
}
else if(datetime.endsWth("/..") || datetime.endsWith("/") {
       datetime = datetime.split("/")[0]   // Double check if "/" works or need "\/"
}
else if(.......)